transform/cat021_v10.go

```diff
@@ -0,0 +1,1162 @@
+package transform
+
+import (
+	"encoding/hex"
+	"errors"
+	"math"
+	"strings"
+
+	"github.com/mokhtarimokhtar/goasterix"
+)
+
+const (
+	BYTESIZE = 8
+)
+
+var ErrTypeUnknown021 = errors.New("[ASTERIX Error CAT021] Message TYPE Unknown")
+
+type WGS84Coordinates struct {
+	Latitude  float32 `json:"latitude,omitempty"`
+	Longitude float32 `json:"longitude,omitempty"`
+}
+
+type GeometricHeight struct {
+	Height      float64 `json:"height,omitempty"`
+	GreaterThan bool    `json:"greaterthan,omitempty"`
+}
+
+type AirSpeed struct {
```

Comment:
Only TrueAirspeed in the spec. Why Airspeed?

Reply:
On page 83 of the spec (https://www.eurocontrol.int/sites/default/files/2021-02/asterix-adsbtr-cat021-part12-v2-5.pdf), the FRNs for both are listed